src/include/pgexporter.h

```diff
@@ -167,11 +199,14 @@ struct configuration
    int number_of_servers;        /**< The number of servers */
    int number_of_users;          /**< The number of users */
    int number_of_admins;         /**< The number of admins */
+   int number_of_metrics;        /**< The number of metrics*/
+
+   char metrics_path[MISC_LENGTH]; /**< The metrics path */
 
    struct server servers[NUMBER_OF_SERVERS];       /**< The servers */
    struct user users[NUMBER_OF_USERS];             /**< The users */
    struct user admins[NUMBER_OF_ADMINS];           /**< The admins */
-   struct prometheus prometheus;                   /**< The Prometheus metrics */
+   struct prometheus prometheus[NUMBER_OF_METRICS];/**< The Prometheus metrics */
```

Comment:
We can look at this - maybe there is just 1 `yaml_queries` in `prometheus` instead... Keep as is for now